### ISSUE TITLE
Setup emits warning `xargs: warning: options --max-args and --replace/-I/-i are mutually exclusive, ignoring previous --max-args value`

### ISSUE THREAD

**Issue by localheinz** [shivammathur/setup-php#712]
**Describe the bug**

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
- `ubuntu-latest` 

**PHP versions**
 - `8.1.16`
 - `8.2.3`
 - `8.3.0-dev`

**To Reproduce**
See [`.github/workflows/ci.yml`](https://github.com/sebastianbergmann/phpunit/blob/08c7e391383eaf8304cdf15b00d48d0c20c4ddc6/.github/workflows/ci.yml#L62-L116) in [`sebastianbergmann/phpunit`](https://github.com/sebastianbergmann/phpunit):

```yaml
# https://help.github.com/en/categories/automating-your-workflow-with-github-actions

on:
  - pull_request
  - push

name: CI

env:
  COMPOSER_ROOT_VERSION: "10.1-dev"

permissions:
  contents: read

jobs:
  unit-tests:
    name: Unit Tests

    runs-on: ${{ matrix.os }}

    env:
      PHP_EXTENSIONS: none, curl, dom, json, libxml, mbstring, openssl, phar, soap, tokenizer, xml, xmlwriter
      PHP_INI_VALUES: memory_limit=-1, assert.exception=1, zend.assertions=1, error_reporting=-1, log_errors_max_len=0, display_errors=On

    continue-on-error: ${{ matrix.experimental }}

    strategy:
      fail-fast: false
      matrix:
        os:
          - ubuntu-latest
          - windows-latest

        php-version:
          - "8.1"
          - "8.2"

        experimental:
          - false

        include:
          - os: ubuntu-latest
            php-version: "8.3"
            experimental: true

          - os: windows-latest
            php-version: "8.3"
            experimental: true

    steps:
      - name: Configure Git to avoid issues with line endings
        if: matrix.os == 'windows-latest'
        run: git config --global core.autocrlf false

      - name: Checkout
        uses: actions/checkout@v3

      - name: Install PHP with extensions
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ matrix.php-version }}
          extensions: ${{ env.PHP_EXTENSIONS }}
          ini-values: ${{ env.PHP_INI_VALUES }}
          tools: none

      - name: Install dependencies with Composer
        run: ./tools/composer update --no-ansi --no-interaction --no-progress

      - name: Run tests with PHPUnit
        run: ./phpunit --testsuite unit
```

**Expected behavior**
Setup fails or does not emit a warning

**Screenshots/Logs**
See 

- https://github.com/sebastianbergmann/phpunit/actions/runs/4428409286/jobs/7767447793#step:4:19
- https://github.com/sebastianbergmann/phpunit/actions/runs/4428409286/jobs/7767448266#step:4:19
- https://github.com/sebastianbergmann/phpunit/actions/runs/4428409286/jobs/7767449201#step:4:19

**Additional context**
n/a

**Are you willing to submit a PR?**
n/a

**Comment by shivammathur** [comment]
Fixed in https://github.com/shivammathur/setup-php/commit/39b4e4a983d4ee3b1f984a81eca6a46994718761, Will be in the next release.

**Comment by shivammathur** [comment]
Fix released in [`2.25.0`](https://github.com/shivammathur/setup-php/releases/tag/2.25.0).

**Comment by localheinz** [comment]
Thank you, @shivammathur!
